lib/event_sourcery/aggregate_root.rb

```diff
@@ -1,5 +1,55 @@
 module EventSourcery
+  #
+  # EventSourcery::AggregateRoot provides a foundation for writing your own AggregateRoot classes.
+  # You can use it by including it in your classes, as show in the example code.
+  #
+  # Excerpt from {https://github.com/envato/event_sourcery/blob/master/docs/core-concepts.md EventSourcery Core Concepts} on Aggregates follows:
+  # === Aggregates and Command Handling
+  #
+  #   An aggregate is a cluster of domain objects that can be treated as a single unit.
+  #   Every transaction is scoped to a single aggregate. An aggregate will have one of its component objects be
+  #   the aggregate root. Any references from outside the aggregate should only go to the aggregate root.
+  #   The root can thus ensure the integrity of the aggregate as a whole.
+  #
+  #   — DDD Aggregate
+  #
+  # Clients execute domain transactions against the system by issuing commands against aggregates.
+  # The web layer will take a command and pass it to the appropriate command handler. The command handler instantiates the aggregate in question and defers to it to process the command.
+  # When created, an aggregate loads its state by loading all events that pertain to it from the event store. The aggregate then processes the command by either rejecting it or accepting it. The aggregate emits events into the event store when it successfully processes a command.
```

Comment:
This bit was snipped from the Core Concepts doc. And it seems that the core concepts have a new and updated section about aggregate roots thats more succinct. So I've replace this with the new updated text.